indra/newview/lluilistener.cpp

```diff
@@ -443,6 +448,11 @@ void LLUIListener::uploadLocalTexture(const LLSD& event) const
     });
 }
 
+LLSD LLUIListener::getGraphicsQuality(LLSD const& request)
```

Comment:
We can use `LLDebugSettings.get('RenderQualityPerformance')` in Lua

Reply:
You're right. It would be cleaner to revert the change I added.

Reply:
https://github.com/secondlife/viewer/pull/3268/commits/ca5b1e4ea7aa8cb012356c44133fa5037d202449

Reply:
ship it